README.md

```diff
@@ -1,4 +1,6 @@
-[buildah](https://www.youtube.com/embed/YVk5NgSiUw8) - a tool which facilitates building OCI container images
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/logos/buildah.svg)
```

Comment:
I think this needs a `master` between `buildah` and `logos` to point to a branch.

Reply:
👍